.github/labels.yaml

```diff
@@ -0,0 +1,6 @@
+# This should enable comments such as "/area hypershift-operator" to add
+# the label to a PR/Issue
+# See https://github.com/jpmcb/prow-github-actions/blob/main/docs/labeling.md
+area:
```

Comment:
Just to clarify: This enable as to use /area via the github action plugin. - Don't we need to define the plugin somewhere as in https://github.com/jpmcb/prow-github-actions#quickstart so GH knows how/what to do with this labels.yaml file?

Then the OWNERS files tells prow where to apply which ones then by default.

This https://github.com/openshift/release/pull/36830 will prevent a PR merging without at least one of them.

Reply:
Sorry, I looked at this first, but found out that prow supports this via the labels plugin without the need for github actions:
https://prow.k8s.io/command-help#kind
It was not clear to me whether this file was still needed, so I left it in.

Reply:
shall we remove the comment then?